.gitignore

```diff
@@ -15,6 +15,7 @@ lib/
 *.iws
 *~
 *.swp
+^dev-support/artifacts
```

Comment:
I don't think we need the `^` character

Also I would add `dev-support/work` here too

Reply:
Right, that is not a regex.